api/src/main/java/io/cloudevents/DataConversionException.java

```diff
@@ -0,0 +1,8 @@
+package io.cloudevents;
+
+public class DataConversionException extends RuntimeException {
+
+    public DataConversionException(String from, String to, Throwable cause) {
```

Comment:
```suggestion
    public DataConversionException(Object from, Object to, Throwable cause) {
```

would allow `throw new DataConversionException(somethingNullable, someOtherNullable, ex)`